tools/cldr-apps/js/src/specialToComponentMap.js

```diff
@@ -10,6 +11,7 @@ const specialToComponentMap = {
   about: AboutPanel,
   retry: WaitingPanel,
   vsummary: VettingSummary,
+  r_vetting_json: DashboardPanel,
   // If no match, end up here
```

Comment:
at some point we might rename "r_vetting_json" to "dashboard" or "dash"

Reply:
Yes, but that would break the non-dojo side, and existing URLs.

Reply:
future todo